client/components/editor/teaching-elements/Assessment/TextSelection/Range.js

```diff
@@ -0,0 +1,60 @@
+import { extractIndex, isBlast } from './helpers';
+import filter from 'lodash/filter';
+import first from 'lodash/first';
+import inRange from 'lodash/inRange';
+import last from 'lodash/last';
+
+const isText = node => node.nodeType === Node.TEXT_NODE;
+const sort = (...values) => values.sort((a, b) => a - b);
+
+export default class Range {
+  constructor(start, end) {
+    [start, end] = sort(start, end);
+    this[0] = start;
+    this[1] = end;
```

Comment:
Well, because you don't use it as array convert it to `this._start` & `this._end` :bowtie: